automat/_methodical.py

```diff
@@ -75,18 +75,30 @@ def _transitionerFromInstance(oself, symbol, automaton):
         setattr(oself, symbol, transitioner)
     return transitioner
 
+def _empty():
+    pass
+_empty() # chase coverage
```

Comment:
I would rather chase this coverage in the unit tests rather than in the import side effects